src/cpp/session/modules/build/SessionBuild.cpp

```diff
@@ -724,6 +724,9 @@ class Build : boost::noncopyable,
       core::system::setenv(&childEnv, "R_BROWSER", "false");
       core::system::setenv(&childEnv, "R_PDFVIEWER", "false");
 
+      // allow cli::style_hyperlink()
+      core::system::setenv(&childEnv, "R_CLI_HYPERLINKS", "true");
```

Comment:
otherwise set the R option `cli.hyperlink = TRUE` ?

Reply:
Setting the associated environment variable in the main session seems reasonable to me.